datumaro/plugins/voc_format/converter.py

```diff
@@ -559,16 +563,17 @@ def _load_categories(self, label_map_source):
                 (label_map_source, ', '.join(t.name for t in LabelmapType)))
 
         # There must always be a label with color (0, 0, 0) at index 0
-        bg_label = find(label_map.items(), lambda x: x[1][0] == (0, 0, 0))
-        if bg_label is not None:
-            bg_label = bg_label[0]
-        else:
-            bg_label = 'background'
-            if bg_label not in label_map:
-                has_colors = any(v[0] is not None for v in label_map.values())
-                color = (0, 0, 0) if has_colors else None
-                label_map[bg_label] = [color, [], []]
-        label_map.move_to_end(bg_label, last=False)
+        if self._add_background:
+            bg_label = find(label_map.items(), lambda x: x[1][0] == (0, 0, 0))
+            if bg_label is not None:
+                bg_label = bg_label[0]
+            else:
+                bg_label = 'background'
+                if bg_label not in label_map:
+                    has_colors = any(v[0] is not None for v in label_map.values())
+                    color = (0, 0, 0) if has_colors else None
+                    label_map[bg_label] = [color, [], []]
+            label_map.move_to_end(bg_label, last=False)
```

Comment:
Please clarify the logic behind this parameter. How is it supposed to work, when there are masks?

Reply:
I fixed one point here. This parameter is only useful when there is no background label. If `True`, we add background, otherwise we don't add it. If there are masks, we need to use `add_background = True` (by default, `True`).

Reply:
> If there are masks, we need to use ...

Who? The user? Code? Do we need to throw errors in specific cases?